aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md

```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
+    [![Screenshot of Choose and Add-on wizard, with Clear D B My S Q L Database selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image4.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image3.png)
 4. Keep the default **Free** plan, change the **NAME** to **IdentityMySQLDatabase**, select the region that is nearest to you, and then click the **Next** arrow at the bottom of the frame:
 
    [Click the following image to expand it. ]
-    [![Illustrates how to change the NAME to IdentityMySQLDatabase and select region.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image6.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image5.png)
+    [![Screenshot of Personalize Add-on dialog, with Free plan highlighted and selected and NAME and REGION selected below. NAME field is filled with Identity My S Q L Database and REGION field is filled with the example West U S.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image6.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image5.png)
 5. Click the **PURCHASE** checkmark to complete the database creation.
 
    [Click the following image to expand it. ]
-    [![Illustrates Review Purchase process with PURCHASE checkmark selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image8.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image7.png)
+    [![Screenshot of Review Purchase dialog with PURCHASE checkmark selected at lower right.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image8.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image7.png)
 6. After your database has been created, you can manage it from the **ADD-ONS** tab in the management portal. To retrieve the connection information for your database, click **CONNECTION INFO** at the bottom of the page:
 
    [Click the following image to expand it. ]
-    [![Illustrates management portal with ADD-ONS tab highlighted at left and CONNECTION INFO highlighted at bottom.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image10.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image9.png)
+    [![Screenshot of management portal with CREDIT STATUS tab selected in top menu, ADD-ONS tab highlighted at left and CONNECTION INFO highlighted at bottom. Identity My S Q L database name is highlighted at right.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image10.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image9.png)
```

Comment:
Too long at 203 characters; consider: "Screenshot of the management portal with the Add Ons tab, the Identify My S Q L database item, and the Connection Info button highlighted in red."